database/migrations/2025_02_17_080713_more_privacy_options.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * SPDX-License-Identifier: MIT
+ * Copyright (c) 2017-2018 Tobias Reich
+ * Copyright (c) 2018-2025 LycheeOrg.
+ */
+
+use App\Models\Extensions\BaseConfigMigration;
+
+return new class() extends BaseConfigMigration {
+	public function getConfigs(): array
+	{
+		return [
+			[
+				'key' => 'exif_disabled_for_all',
+				'value' => '0',
+				'cat' => 'Mod Privacy',
+				'type_range' => self::BOOL,
+				'description' => 'Disable details and overlay panels in front-end.',
+				'details' => '<span class="pi pi-exclamation-triangle text-orange-500"></span> This will not remove the data from the API end-point.',
+				'is_secret' => false,
+				'level' => 1,
+			],
+			[
+				'key' => 'file_name_hidden',
+				'value' => '0',
+				'cat' => 'Mod Privacy',
+				'type_range' => self::BOOL,
+				'description' => 'Do not show the photo title to anonymous users.',
+				'details' => 'Logged in user will still have access to the title.',
+				'is_secret' => true,
```

Comment:
I don't see a problem with it, but why secret?

Reply:
oversight, there is no need for that to be secret.

Reply:
fixed : [f3bb7da](https://github.com/LycheeOrg/Lychee/pull/3014/commits/f3bb7da1c06078c9756e10784cf48a299db7408f)